index.js

```diff
@@ -81,6 +81,10 @@ is.promise = x => {
 		);
 };
 
+is.generator = x => x && typeof x.next === 'function' && typeof x.throw === 'function';
```

Comment:
Should be documented that this is just a naive check, like done with `is.promise`.

Reply:
We can actually make the check a little bit better:

```js
is.generator = x => x && typeof x[Symbol.iterator] === 'function' && typeof x.next === 'function' && typeof x.throw === 'function';
```

Reply:
You can use `is.iterable` here

Reply:
No, by definition, all generators are iterators. See [here](https://stackoverflow.com/a/19660350) why just checking for an `iterable` isn't enough in this case.

Reply:
I meant that some of that check is already done by `is.iterable`, not all of it. F.e the symbol check

Reply:
Yes, let's use `is.iterable` here.

Reply:
@gioragutt @sindresorhus coolio. @gioragutt I thought you meant just check if the object is iterable. Whoopsie. 🙊